### PR TITLE
Added UI skeleton of Poll screen

### LINKED ISSUE #14: Poll screen skeleton

### PR DESCRIPTION
**Pull request by kennygperez**
My dumb butt didn't make a branch for this change...

Fixed #14

### CODE REVIEW

**Comment by kennygperez** [comment]
@weining-li

File: mobile/src/components/Radios.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+export const Radios: FunctionComponent<RadioProps> = ({
+	options,
+	onSelect,
+}) => {
+	const [selectedOption, setSelectedOption] = useState<null | Option>(null);
+
+	const isSelected = (): boolean => {
+		return false;
+	};
```
Comment (weining-li) [inline mobile/src/components/Radios.tsx:59]: :thinking:

**Review by weining-li — APPROVED** [review]
I mean I guess it's good?

File: mobile/src/components/Radios.tsx (hunk 1 above)
Comment (ljoks) [reply mobile/src/components/Radios.tsx:59]: 🧠

**Review by ljoks — APPROVED** [review]

**Pull request merged**